xonsh/aliases.py

```diff
@@ -820,6 +821,11 @@ def sudo(args):
         default_aliases["grep"] = ["grep", "--color=auto"]
         default_aliases["egrep"] = ["egrep", "--color=auto"]
         default_aliases["fgrep"] = ["fgrep", "--color=auto"]
+    elif ON_OPENBSD:
+        default_aliases["grep"] = ["grep"]
+        default_aliases["egrep"] = ["egrep"]
+        default_aliases["fgrep"] = ["fgrep"]
+        default_aliases["ls"] = ["ls"]
```

Comment:
Why do you introduce aliases which do not modify their respective command in any way?
If you don't want any aliases on OpenBSD, just say `pass` here.

Reply:
Thanks!